binaries/opae.io/opae/io/utils.py

```diff
@@ -186,6 +200,19 @@ def vfio_release(pci_addr):
         return
 
     print('Releasing {} from vfio-pci'.format(msg))
+
+    # On Linux kernel >= 3.15, clear driver_override before
+    # unbinding vfio-pci driver, which ensures subsequent
+    # binding to another, previously bound driver succeeds.
+    driver_override = '/sys/bus/pci/devices/{}/driver_override'.format(pci_addr)
+    if os.path.exists(driver_override):
+        try:
+            with open(driver_override, 'w') as outf:
+                outf.write('\n')
+        except OSError as exc:
+            LOG.error('Cannot clear driver_override for {}: {}'.format(msg, exc))
+            return
+
```

Comment:
I believe these comments and code snippets are based on dpdk_devicebind.py.  Please ensure both bits of code and comments are copyrighted by Intel.